Initialtask/src/Entity/Product.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ProductRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="tblProductData")
+ * @ORM\Entity(repositoryClass=ProductRepository::class)
+ */
+class Product
+{
+    public const MIN_COST = 5;
+
+    public const MAX_COST = 1000;
+
+    public const MIN_STOCK = 10;
+
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(name="intProductDataId", type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(name="strProductName", type="string", length=50)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(name="strProductCode", type="string", length=10, unique=true)
+     */
+    private $code;
+
+    /**
+     * @ORM\Column(name="strProductDesc", type="string", length=255)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(name="intStock", type="integer", nullable=true)
+     */
+    private $stock;
+
+    /**
+     * @ORM\Column(name="floatCost", type="float", nullable=true)
+     */
+    private $cost;
+
+    /**
+     * @ORM\Column(name="dtmDiscontinued", type="datetime", nullable=true)
+     */
+    private $discontinued;
+
+    /**
+     * @ORM\Column(name="dtmAdded", type="datetime", nullable=true)
```

Comment:
Use symfony validation annotation and validator to check if created entity is valid